strings/censor.py

```diff
@@ -0,0 +1,14 @@
+def censor(txt, word):
+  txt_lst = txt.split()
+  #here we loop through a list which consists of words from the text we should censor
+  for word1 in txt_lst:
+    if word1 == word:
+      txt_lst[txt_lst.index(word1)] = "*" * len(word1)
```

Comment:
You could make this a little more efficient by using ```for index, word1 in enumerate(txt_lst)```. 
That way you already have the index and don't need to call ```txt_lst.index(word1)```. Instead you just use the variable ```index```.